README.md

```diff
@@ -391,6 +392,19 @@ const { product } = await getProduct({
 })
 ```
 
+## Checkout
+
+> The checkout only works on **production** and with a custom domain
+
+The recommended method is the [Embedded Checkout](https://developer.bigcommerce.com/api-docs/storefronts/embedded-checkout/embedded-checkout-tutorial), follow the tutorial to create a channel and a site. Notes:
+
+- The channel should be of type `storefront`
+- The site url must be your production url (e.g: https://mystorefront.com)
+- This package takes care of the cart and redirect links creation
+- Your bigcommerce store must be a subdomain of your headless store (eg: https://bc.mystorefront.com)
```

Comment:
@jorgemasta Looks like this didn't get updated to "storefront" for both usages of "store".